doc/format.rst

```diff
@@ -597,6 +597,24 @@ Other points to keep in mind
   (i.e. scalar types, sequence types), those arguments can be documented
   with type `array_like`.
 
+* Links : If you need to include hyperlinks in your docstring, note that
+  some docstring sections are not parsed as standard reST, and in these
+  sections, numpydoc may become confused by hyperlink targets such as::
+
+      .. _Example: http://www.example.com
+
+  If the Sphinx build issues a warning of the form
+  ``WARNING: Unknown target name: "example"``, then that is what is happening.
+  There are two possible workarounds.  First, the inline form can be used::
+
+      `Example <http://www.example.com>`_
```

Comment:
Should this be ``` `Example <http://www.example.com`__ ``` ?

Reply:
I'm pretty sure that inline hyperlink targets need a closing `>` and one underscore, not two.  Please see http://docutils.sourceforge.net/docs/user/rst/quickref.html#hyperlink-targets

Reply:
The missing `>` was a typo; I was concerned with what happens when you have similar inline references in different docstrings.